td.vue/src/service/x6/ports.js

```diff
@@ -0,0 +1,102 @@
+import { Graph } from '@antv/x6';
+
+export const ports = {
+    groups: {
+        top: {
+            position: 'top',
+            attrs: {
+                circle: {
+                    r: 4,
+                    magnet: true,
+                    stroke: '#5F95FF',
+                    strokeWidth: 1,
+                    fill: '#fff',
+                    style: {
+                        visibility: 'hidden'
+                    }
+                }
+            }
+        },
+        right: {
+            position: 'right',
+            attrs: {
+                circle: {
+                    r: 4,
+                    magnet: true,
+                    stroke: '#5F95FF',
+                    strokeWidth: 1,
+                    fill: '#fff',
+                    style: {
+                        visibility: 'hidden'
+                    }
+                }
+            }
+        },
+        bottom: {
+            position: 'bottom',
+            attrs: {
+                circle: {
+                    r: 4,
+                    magnet: true,
+                    stroke: '#5F95FF',
+                    strokeWidth: 1,
+                    fill: '#fff',
+                    style: {
+                        visibility: 'hidden'
+                    }
+                }
+            }
+        },
+        left: {
+            position: 'left',
+            attrs: {
+                circle: {
+                    r: 4,
+                    magnet: true,
+                    stroke: '#5F95FF',
+                    strokeWidth: 1,
+                    fill: '#fff',
+                    style: {
+                        visibility: 'hidden'
+                    }
+                }
+            }
+        }
+    },
+    items: [
+        {
+            group: 'top'
+        },
+        {
+            group: 'right'
+        },
+        {
+            group: 'bottom'
+        },
+        {
+            group: 'left'
+        }
+    ]
+};
+
+Graph.registerNode(
```

Comment:
I think this was left over from testing, we don't reference customRect anywhere.